eth/api_tracer.go

```diff
@@ -805,7 +805,15 @@ func (api *PrivateDebugAPI) computeTxEnv(blockHash common.Hash, txIndex int, ree
 
 	for idx, tx := range block.Transactions() {
 		// Assemble the transaction call message and return if the requested offset
-		msg, _ := tx.AsMessage(signer)
+		var msg core.Message
+		if !vm.UsingOVM {
+			msg, _ = tx.AsMessage(signer)
+		} else {
+			msg, err = core.AsOvmMessage(tx, signer, common.HexToAddress("0x4200000000000000000000000000000000000005"))
```

Comment:
Bleh idk where to get this address from within this context.

Reply:
I feel like its a code smell that we hard code this here but its unlikely to change. The way to pass it through it to have a method on the `Backend` that returns the address